website/utils/plugins.js

```diff
@@ -4,12 +4,24 @@ const matter = require("gray-matter");
 const title = require("title");
 const pluginTitle = (plugin) => title(plugin.replace(/-/g, " "));
 
-const getData = (pluginsDir, plugin) => {
+const readPluginOverview = (pluginsDir, plugin) => {
```

Comment:
I missed this in https://github.com/cloudquery/cloudquery/pull/14484. We read the `overview.mdx` file to get the title and stage for https://www.cloudquery.io/docs/plugins/sources/overview and https://www.cloudquery.io/docs/plugins/destinations/overview